app/services/set_public_id_for_qualification.rb

```diff
@@ -0,0 +1,33 @@
+class SetPublicIdForQualification
```

Comment:
Second thoughts about putting this in a service, it feels like it belongs on the model. That way I can call it from the model whenever a qualification is updated